Automattic/simplenote-macos
# Refreshing Notes List: Mark 0

### Details:
- Adds Simplenote iOS **Preview String Generation** tooling. [PR Here!](https://github.com/Automattic/simplenote-ios/pull/628)
- Wraps `kPreviewLinesPref` as an **Options property**

**Yes!** 99% of this PR is code already shipped in the iOS app.

This is the first PR of a multipart approach. For that reason, `bodyPreview` isn't wired right here!

@guarani **Thanks A LOT in advance sir!!!**
Ref. #458

### Test: List Lines
1. Install `develop`
2. Sign into your account 
3. Click over `View > Notes List > Note Display > Comfy`
4. Verify the Notes List is now condensed!
5. Install this branch on top

- [ ] Verify the Notes List still looks condensed!
- [ ] Verify the Unit Tests pass

### Release
These changes do not require release notes.
